src/NuGet.Core/NuGet.ProjectModel/PackageSpecUtility.cs

```diff
@@ -73,17 +73,26 @@ public static bool IsSnapshotVersion(string version)
                 packageTargetFallback,
                 assetTargetFallback);
 
-            if (assetTargetFallback?.Any() == true)
+            var hasAssetTargetFallback = assetTargetFallback?.Any() == true;
```

Comment:
It could be null...so that turns the type to bool?. 
Simpler to deal with that here rather than where we use the bool.